### DISCUSSION TITLE
How to turn off entry title editing

### DISCUSSION THREAD

**Discussion by cetfu** [dlemmermann/CalendarFX#156] [Q&A]
Hello, how can I turn off entry title editing?

**Comment by Qshihiku** [comment]
Can this be used for maps or not? If possible, can you cooperate with me so that you can discuss it, you can contact me directly via WhatsApp: 082122968546

**Comment by Qshihiku** [comment]
Can this be used for maps or not? If possible, can you cooperate with me so that you can discuss it, you can contact me directly via WhatsApp: 082122968546
